### ISSUE TITLE
Essentials

### ISSUE THREAD

**Issue by TheiBa** [systemspharmacology/systemspharmacology.github.io#15]
Wording should remain? I mean, I am ok to hear that even your mum loves it, but should this wording stay? Do you want to propose something else or are you expecting input?

**Comment by msevestre** [comment]
Inputs is required for each and every section of the site. The text added here is at best placeholder and well sometime not even that good :)

**Comment by TheiBa** [comment]
Ah, ok … but I think that is the only place left then? And why not place something for further discussion and review? 

Proposal: 

Downlod: Download and installation only requires to accept GPLv2 license and will allow you to explore the full functionality of the OSP suite. 

Forum: Ask questions, get and provide answers or input. Also stay up-to-date on latest announcements and surveys. 

Documentation: 500 pages of software manual including additional documentation. Missing something or prefer wiki style – feedback and input welcome!

**Comment by msevestre** [comment]
Ok good Idea I have updated the site.

My comments:
Dowload: The text is technically not accurate. You don't have to require GPLv2 (plus it;s not even that it's that plus amendment). I think this needs to be rephrase (not sure I would mention the license at all in the summary here)

Documentation:
Lol. After that, I really don't want to read it. "500 pages of software manual" is not an enthusiastic description :)

**Comment by TheiBa** [comment]
I am fine with alternative proposals, as long as they don’t concern your mum ;-) 

My understanding was that by installation you accept GPLv2 terms, and I though this is kind of important information, i.e. really open source, free use for almost whatever you want. But you can also put a different emphasis here. 

Doc, similar. Self-explaining software with context-dependent help, and if you love reading – enjoy 500 pages of manual amongst others. Or similar. My wording was maybe not enthusiastic, but I still believe the message is relevant: this is not a poorly documented, find-your-own way home-brewn solution for techies only, but comes with “all” you need to operate it as an interested scientist / non-developer.

**Comment by msevestre** [comment]
FYI: In terms of documentation, my vision is to move to a version where each pages is hosted on github as a wiki. (in a separate repo obviously). We would create a website out of the doc that one could navigate (using https://dotnet.github.io/docfx/index.html... this is what microsoft is using for its documentation in .NET)
The beauty is that people could submit changes directly via github and so to speak, the doc would grow organically.

**Comment by msevestre** [comment]
Example of (very well done, we will never achieve that level of pro) documentation
https://docs.microsoft.com/

**Comment by msevestre** [comment]
https://docs.microsoft.com/en-us/dotnet/core/tools/cli-msbuild-architecture

a page with many contributors and an edit button to  submit changes

**Comment by TheiBa** [comment]
yes, nice to go that way – but the 500 pages is what is there and may not only become available in 2 years ;-) but if available in a few days, at least as a start, we can directly put it …

**Comment by msevestre** [comment]
absolutely. I just wanted to let you know what I'd like to see. But this
definitely won't be available in the near future

**Comment by msevestre** [comment]
@Yuri05 Text needs to be reviewed
